app/components/rdf-input-fields/concept-scheme-multi-selector.js

```diff
@@ -0,0 +1,134 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import {
+  triplesForPath,
+  updateSimpleFormValue,
+} from '@lblod/submission-form-helpers';
+import InputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/input-field';
+import { restartableTask, timeout } from 'ember-concurrency';
```

Comment:
this might break with the upgrade of ember concurrency, give it a look before you merge please